common.go

```diff
@@ -0,0 +1,104 @@
+package common
```

Comment:
Kinda weird to have a `common` package at the root of the project (i.e. github.com/KingDerp/memoryLane). I'd consider moving this under `pkg/common` or `pkg/webutil` or something.

Otherwise, this is a very cleanly organized file and set of functions. great jorb!